README.md

```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
 
 For an introduction to the project's organization and key repos, check out the [Technical Overview](https://tessel.gitbooks.io/t2-docs/content/Debugging/Technical_Overview.html).
 
-## How can I get involved?
+# Current goals
 
-### Quick start
+On September 2, 2016, the Steering Committee set year-long goals. They are organized into two overarching goals, each with several supporting goals:
+
+1. **Grow the community for development, inclusion, accessibility**
+  * [ ] Build effective working groups that can complete their tasks with measurable results. Celebrate the work of each of the working groups as goals are accomplished
+  * [ ] Upgrade our documentation. Build more fritzing examples, API prototypes, and call out features that are currently missing from the docs
+  * [ ] Update the tessel.io website to more accurately represent the project and its plans (https://github.com/tessel/tessel.io/issues/102)
+2. **Demonstrate projects and create new tools that prove Tessel is a production-ready (robust, reliable, and extensible) platform for modern hardware development**
+  * [ ] Create a guide to production-scale deployment of a Tessel project
+  * [ ] Research user needs: What do people need to use T2 in production? Why aren't they using it currently?
+  * [ ] Build Reach (https://github.com/tessel/project/issues/142)
+  * [ ] Get to JavaScript parity with Rust API and documentation (https://github.com/tessel/tessel-rust). Figure out JS-Rust inter-exection
+  * [ ] Investigate and possibly execute a Tessel hardware upgrade (Tessel 2.1) for more RAM and Flash. This should legitimize the platform and build out the capabilities so there is less user experience friction
+
+## Working groups
+
+To enable the above goals, the Tessel Project is establishing working groups.
```

Comment:
Change "is establishing" to something that will be current. "The Tessel Project forms working groups to accomplish our year-long goals."